SECURITY.md

```diff
@@ -0,0 +1,9 @@
+## Report a security issue
+
+The faslty-template-rust-default project team welcomes security reports and is committed to providing prompt attention to security issues. Security issues should be reported privately via [Fastly’s security issue reporting process](https://www.fastly.com/security/report-security-issue).
+
+## Security advisories
+
+Remediation of security vulnerabilities is prioritized by the project team. The project team endeavors to coordinate remediation with third-party stakeholders, and is committed to transparency in the disclosure process. The team announces security issues via $RELEASE-ANNOUNCEMENT-E.G.-GITHUB-RELEASE as well as $ECOSYSTEM-SPECIFIC-ADVISORY-DATABASE-EG-RUSTSEC on a best-effort basis.
```

Comment:
Are these variables meant to be like this in the final output?

Reply:
😱 Good catch, which means we also have these in the default template too.